samples/detect.js

```diff
@@ -52,43 +68,43 @@ function detectTextIntent(projectId, sessionId, queries, languageCode) {
       },
     };
 
-    if (!promise) {
-      // First query.
-      console.log(`Sending query "${query}"`);
-      promise = sessionClient.detectIntent(request);
-    } else {
-      promise = promise.then(responses => {
-        console.log('Detected intent');
-        const response = responses[0];
-        logQueryResult(sessionClient, response.queryResult);
-
-        // Use output contexts as input contexts for the next query.
-        response.queryResult.outputContexts.forEach(context => {
-          // There is a bug in gRPC that the returned google.protobuf.Struct
-          // value contains fields with value of null, which causes error
-          // when encoding it back. Converting to JSON and back to proto
-          // removes those values.
-          context.parameters = struct.encode(struct.decode(context.parameters));
-        });
-        request.queryParams = {
-          contexts: response.queryResult.outputContexts,
-        };
-
-        console.log(`Sending query "${query}"`);
-        return sessionClient.detectIntent(request);
-      });
+    // If we have contexts from previous interactions with the bot add them here
```

Comment:
I think we can remove this from the quickstart.

Reply:
👌